### PR TITLE
about a first iteration

### PR DESCRIPTION
**Pull request by klees**

### CODE REVIEW

File: CONCEPT.md [hunk 1]
```diff
@@ -354,16 +369,18 @@ Edge Cases:
...
 a "MessageBox > Info" stating that the “Learning Progress” will be available when
 the “Line Item” is available.
+(Das verstehe ich glaube ich nicht mehr. Wir legen doch immer ein Line Item an, jetzt? - RK)
 
 The learning progress tab provides screens to optionally enable and access the
 learning progress of ILIAS users of the tool. It is visible/accessible,
-if the tool supports the "Assignment and Grade Services Specification".
-Furthermore, the regular access checks apply for all subordinted views with this tab.
+if the tool supports the "Assignment and Grade Services Specification" (Ich glaube, das
+brauchen wir nicht, Tools müssen das supporten? - RK).
```
Comment (mjansenDatabay) [inline CONCEPT.md]: Das war aber doch AFAIR komplett optional, oder nicht?

File: CONCEPT.md [hunk 2]
```diff
@@ -354,16 +369,18 @@ Edge Cases:
 * Localized Formatting: Test with users in different timezones
   (e.g., "Europe/Berlin", "GMT") to confirm that datetime formatting adapts accordingly.
 
-#### Learning Progress Tab
+#### Learning Progress Tab (I1)
 
 If no “Line Item” is available, the user is shown a corresponding message in
 a "MessageBox > Info" stating that the “Learning Progress” will be available when
 the “Line Item” is available.
+(Das verstehe ich glaube ich nicht mehr. Wir legen doch immer ein Line Item an, jetzt? - RK)
```
Comment (mjansenDatabay) [inline CONCEPT.md]: Ja, wir haben also erst mal keins oder eins. Wenn das Tool gar keinen "AGS" unterstützt (s.u.), was machen wir dann?

Comment (mjansenDatabay) [inline CONCEPT.md:377]: Ja, wir haben also erst mal keins oder eins. Wenn das Tool gar keinen "AGS" unterstützt (s. u.), was machen wir dann?

Comment (mjansenDatabay) [inline CONCEPT.md:377]: Ja, wir haben also erst mal keins oder eins. Wenn das Tool gar keinen "AGS" unterstützt (s. u.), was machen wir dann?

File: CONCEPT.md [hunk 3]
```diff
@@ -464,11 +482,13 @@ Expected Data Presentation Format:
 | `2024-11-22 15:32:12` | `DeepLinkingResponse` | `{ "content_items": [...], "userId": "456" }` |
 | `2024-11-22 15:30:00` | `LaunchRequest`       | `{ "userId": "123", "role": "Student" }`      |
 
-The folliwing filters are provided:
+The following filters are provided:
 
 * Date Range (type: duration input / default values: start = \[NOW - 1 week\], end = \[NOW\])
 
-#### Metadata (Discovery/Searchability)
+(Vielleicht hier noch den "angemeldeten User" aufnehmen? - RK)
```
Comment (mjansenDatabay) [inline CONCEPT.md:486]: Siehe #6
